.changeset/3198f9fa/changes.json

```diff
@@ -0,0 +1,4 @@
+{
+  "releases": [{ "name": "@brisk-docs/website", "type": "patch" }],
```

Comment:
Just a note - since this breaks existing implementations of the package, this should be a MAJOR change, not a patch.

Because we are not doing major releases yet, this should be adjusted to be a minor change.

Reply:
- [ ] Update from patch change to minor change.
- [ ] Modify changeset message in the `changeset.md` to explain necessary changes to make when upgrading.